src/attributes/components/AttributeList/AttributeList.tsx

```diff
@@ -83,23 +83,39 @@ const AttributeList: React.StatelessComponent<AttributeListProps> = ({
         toolbar={toolbar}
       >
         <TableCell className={classes.colSlug}>
-          {i18n.t("Attribute Code", { context: "attribute slug" })}
+          <FormattedMessage
+            defaultMessage="Attribute Code"
+            description="attribute list: slug column header"
```

Comment:
We should be describing the meaning and context, not placement. The translation will be the same whether "Attribute Code" is used as an input label or a column header.